src/util/util_vector.cpp

```diff
@@ -0,0 +1,9 @@
+#include "util_vector.h"
```

Comment:
If this file is empty anyway, please remove it.

Reply:
I rebased this commit to do that before and forgot to actually make the change to the commit 🐸 I'll fix this, yea